monitoring/uss_qualifier/scd/reports.py

```diff
@@ -57,28 +58,49 @@ class Interaction(ImplicitDict):
     interaction_id: InteractionID
     """ID of this interaction (used to refer to this interaction from an issue)"""
 
-    test_id: str
-    """ID of test for which this interaction was performed"""
+    context: AutomatedTestContext
+    """Context in which this interaction was performed"""
 
     test_step: int
-    """Step of test for which this interaction was performed"""
+    """Step of test for which this interaction was performed.
+    0-based indexed.
+    Special values:
+     * -1: capture occurred during test initialization
```

Comment:
Instead of encoding two different things in one field, could we break them out separately?  The `test_step` field could become `step_index` (or something that could potentially be used in initialization and cleanup also), and then we could add a `test_phase` field indicating `Initialization`, `Test`, and `Cleanup`.

Reply:
Sounds great. done.